support/config/constants.go

```diff
@@ -50,11 +50,38 @@ const (
 
 	AuditWebhookService = "audit-webhook"
 
+	// DefaultMachineNetwork is the default network CIDR for the machine network.
+	DefaultMachineNetwork = "10.0.0.0/16"
+)
+
+// Managed Azure Related Constants
+const (
 	// AROHCPKeyVaultManagedIdentityClientID captures the client ID of the managed identity created on an ARO HCP
 	// management cluster. This managed identity is used to pull secrets and certificates out of Azure Key Vaults in the
 	// management cluster's resource group in Azure.
 	AROHCPKeyVaultManagedIdentityClientID = "ARO_HCP_KEY_VAULT_USER_CLIENT_ID"
 
-	// DefaultMachineNetwork is the default network CIDR for the machine network.
-	DefaultMachineNetwork = "10.0.0.0/16"
+	ManagedAzureClientIdEnvVarKey        = "ARO_HCP_MI_CLIENT_ID"
+	ManagedAzureTenantIdEnvVarKey        = "ARO_HCP_TENANT_ID"
+	ManagedAzureCertificatePathEnvVarKey = "ARO_HCP_CLIENT_CERTIFICATE_PATH"
+	ManagedAzureCertificateMountPath     = "/mnt/certs"
+	ManagedAzureCertificatePath          = "/mnt/certs/"
+	ManagedAzureSecretsStoreCSIDriver    = "secrets-store.csi.k8s.io"
+	ManagedAzureSecretProviderClass      = "secretProviderClass"
+
+	ManagedAzureCPOSecretProviderClassName                = "managed-azure-cpo"
```

Comment:
Most of these are not used yet but will be. I wanted to include them here now to avoid rebase issues with the other related PRs.